content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -168,136 +196,142 @@ For example, if there are 1,253 pods on the cluster and the client wants to rece
        "continue": "ENCODED_CONTINUE_TOKEN_2",
        ...
      },
-     "items": [...] // returns pods 501-1000
+     "items": […] // returns Pods 501-1000
    }
    ```
 
-3. Continue the previous call, retrieving the last 253 pods.
+3. Continue the previous call, retrieving the remaining 253 pods.
 
-   ```console
-   GET /api/v1/pods?limit=500&continue=ENCODED_CONTINUE_TOKEN_2
-   ---
-   200 OK
+   ```
+   GET /api/v1/pods?limit=500&continue=ENCODED_CONTINUE_TOKEN_2 HTTP/1.1
+   …more headers…
+   ```
+   ```
+   HTTP/1.1 200 OK
    Content-Type: application/json
 
    {
      "kind": "PodList",
      "apiVersion": "v1",
      "metadata": {
        "resourceVersion":"10245",
-       "continue": "", // continue token is empty because we have reached the end of the list
+       "continue": "", // continue token is empty because this is the end of the list
        ...
      },
-     "items": [...] // returns pods 1001-1253
+     "items": […] // returns Pods 1001-1253
    }
    ```
 
-Note that the `resourceVersion` of the list remains constant across each request, indicating the server is showing us a consistent snapshot of the pods. Pods that are created, updated, or deleted after version `10245` would not be shown unless the user makes a list request without the `continue` token.  This allows clients to break large requests into smaller chunks and then perform a watch operation on the full set without missing any updates.
+Notice that the `resourceVersion` of the PodList remains constant across each response, indicating thay the control plane providing a consistent snapshot of the cluster's Pods. Pods that are created, updated, or deleted after version `10245` would not be reflected in these responses (the client could however make a new List request at any time). This transaction-like behavior allows clients to break large requests into smaller chunks and then perform a Watch operation on the full set, without missing any updates.
 
+## Alternate representations of resources
 
-## Receiving resources as Tables
+### Receiving resources as tables
 
-`kubectl get` is a simple tabular representation of one or more instances of a particular resource type. In the past, clients were required to reproduce the tabular and describe output implemented in `kubectl` to perform simple lists of objects.
-A few limitations of that approach include non-trivial logic when dealing with certain objects. Additionally, types provided by API aggregation or third party resources are not known at compile time. This means that generic implementations had to be in place for types unrecognized by a client.
+By default, `kubectl get <kind>` prints a simple tabular representation of one or more instances of a particular resource type.
+It's useful for other API clients to be able to reproduce the same kind of formatted, tabular representation for a resource; this helps make user interfaces consistent.
```

Comment:
nit: Is line 232 needed? 
Are you indicating that the client is responsible for formatting the server's response into a table format, not the server OR that: When writing an API client, consider how you want to display the returned resources.

Reply:
I've omitted this change from the latest revision.